package.json

```diff
@@ -11,16 +11,7 @@
     "/bin",
     "/npm-shrinkwrap.json",
     "/scripts/postinstall.js",
-    "/src/**/*.js",
-    "/src/**/types.d.ts",
-    "/src/**/*.sh",
```

Comment:
I think you need to do something special about the shell files, and the `function-template` files.

Reply:
oh yea forgot to set it in draft mode